.github/workflows/transifex_push.yml

```diff
@@ -0,0 +1,60 @@
+name: "Transifex Push"
+
+on:
+  workflow_call:
+    secrets:
+      tx_token:
+        required: true
+    inputs:
+      branch:
+        required: false
+        default: 'main'
+        type: 'string'
+
+jobs:
+  ci:
+    name: "Translations"
+    runs-on: ubuntu-latest
+    container:
+      image: "ghcr.io/glpi-project/plugin-builder"
+    steps:
+
+    ##########################################
+    # Checkout repository                    #
+    ##########################################
+    - name: "Checkout"
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ inputs.branch }}
```

Comment:
I think we could simplify it by using the current branch that triggered the workflow (that is the default value).

```suggestion
```

The `inputs:` instruction could then be removed.